tests/integration/test_failing_tasks.py

```diff
@@ -18,6 +18,17 @@ def test_operations_marked_failed_after_failing(
     def no_retry_task(operation_id):
         raise Exception()
 
+    @huey.task()
+    def semaphore_task(operation_id):
+        # this task deletes our operation
+        # so tests will fail if the task failure logic
+        # doesn't abort the rest of the pipeline
+
+        with no_context_app.app_context():
```

Comment:
🤔 "with no context app, call the app context..." Reading this out loud was entertaining. 😄